source/storing-data/provenance.rst

```diff
@@ -2,3 +2,352 @@ Decentralized Provenance Tracking
+captured provenance data is stored within the Archive as well. (Saving as an archive 
+happens automatically with ``q2cli``, and manually with the Artifact API.) For
+reference, :ref:`provenance-structure` contains a detailed discussion of the
+file structure which holds provenance metadata.
+
+Why Capture Provenance Data?
+----------------------------
+
+Provenance gives QIIME 2 users, paper reviewers, and developers valuable
+tools for zero-cost documentation, study validation and reproduction,
+workflow pipelining, and software maintenance and repair.
+
+Among the benefits of this model are:
+
+- Analyses are *fully* reproducible.
+- Analyses self-document, reducing the need for investigator notetaking. For example, q2view produces directed provenance graphs. QIIME 2 Artifacts bring their citations with them. Methods-section text could theoretically be generated from a collection of QIIME 2 Artifacts.
+- Analyses are replay-able. The QIIME 2 team is developing functionality to generate executable scripts from prior results, simplifying the repetition of analyses.
+- In the unlikely event of a data integrity bug, problematic combinations of hardware, environment, Action, and parameters can be investigated effectively. Impacted results can be programatically identified, and could be programatically correctable in some cases.
+
+By capturing provenance metadata at the level of Actions/Results, QIIME 2
+provenance is both host- and interface-agnostic. In other words, a QIIME 2
+analysis can be performed across various host systems, using whatever interfaces
+the user prefers, without compromising the validity of the analysis or the
+provenance. The Result of every step in the analysis contains its own
+unique history.
+
+What Provenance Data is Captured?
+---------------------------------
+
+In order to focus on provenance data, we will consider a relatively simple QIIME 2
+Archive (``.qza``) structure, with limited non-provenance content. Below the
+outer :term:`UUID` directory, this :term:`Artifact` holds the data it
+produced in a ``data`` directory (:ref:`data-goes-in-data`), and a few "clerical"
+files treated at greater length in :doc:`/storing-data/archive`.
+
+.. figure:: ../img/prov_whole_archive.svg
+   :alt: Simplified representation of all files within one Archive, emphasizing how an Archive holds provenance for an arbitrary number of Actions
+
+All that's left to discuss is the ``provenance/`` directory. In the diagram
+above, we use a wiggly blue "multiple-files" icon to represent the collection of
+provenance data associated with one single QIIME 2 action. When this icon appears
+directly within ``provenance/`` the files describe the "current" :term:`Result`.
+All remaining icons appear within the ``artifacts/`` subdirectory. These file
+collections describe all "parent" Results used in the creation of the current Result,
+and are housed in directories named with their respective UUIDs.
+
+.. figure:: ../img/prov_abbreviation.svg
```

Comment:
Great suggestion - these are ready-made icons, and I didn't even notice the flat/bordered style differences. I've updated all of the file icons to flat color. The wiggles look goofy like that, so they've kept their borders.

On which note, unless you feel strongly about it, I'm going to stick with the wigglies. I don't love that I'd be representing both files and directories as a "stack of files", it looks sloppy when I mix the two, and I'd need to add an opaque backing to the file icons to stack them up without looking super eww. Feels like a violation of the juice/squeeze principle right now.

Reply:
> I don't love that I'd be representing both files and directories as a "stack of files", it looks sloppy when I mix the two

The "stack of files" icon convention is really common. While it might be weird mixing the two, it is something that most people have been exposed to in other tools/applications. With that said though, I don't feel particularly strongly about the representation you choose, it just wasn't clear to me on first read that I was actually looking at an abstraction of "multiple files/dirs" (since my brain is wired to see the stack-o-files icon). Once I got to the "wiggly" note, then it was more clear. Also, you should drop the word "wiggly" here IMO and come up with a better name for that icon.

Reply:
Thinking a bit more on this, I think what threw me off when I first saw the wiggly blue icon is the actual "wiggle" line at the bottom. I thought maybe that meant something important to the icon, otherwise why include it? I guess its not clear to me why it's not "just" a stack of rectangles.

Reply:
But, this is all getting into existential crisis territory, no need to do anything besides my request above to remove the word "wiggly" from the description, especially if this is from some pre-made icon pack.

Reply:
Awesome. While working on the other PR, I added an explicit label to all instances of the wiggly files icon. I ported that over here when I switched to flat color file icons. Hopefully that helps. :crossed_fingers: